src/main/java/org/candlepin/subscriptions/subscription/SubscriptionTaskQueueConfiguration.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2021 Red Hat, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * Red Hat trademarks are not licensed under GPLv3. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+package org.candlepin.subscriptions.subscription;
+
+import org.candlepin.subscriptions.task.TaskQueueProperties;
+
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * Task queue configuration for Subscription tasks.
+ */
+@Configuration
+public class SubscriptionTaskQueueConfiguration {
```

Comment:
Please fold this into `CapacityIngressConfiguration`. capacity-ingress is the profile for the worker that currently handles requests from subscription-conduit; I'd rather have that deployment refactored over time to handle all subscription syncing work than introduce yet another deployment type.